src/rules/enforce-console-error.ts

```
@@ -109,40 +129,69 @@ export const enforceConsoleError = createRule<[], MessageIds>({
+        >();
 
         // Group open calls by function
         openCalls.forEach(({ node, severity, functionScope }) => {
           if (!functionGroups.has(functionScope)) {
-            functionGroups.set(functionScope, { openCalls: [], consoleCalls: [] });
+            functionGroups.set(functionScope, {
+              openCalls: [],
+              consoleCalls: [],
+            });
           }
           functionGroups.get(functionScope)!.openCalls.push({ node, severity });
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)